Jenkinsfile-insertions

```diff
@@ -0,0 +1,88 @@
+import groovy.json.JsonSlurper
+
+pipeline{
+	agent any
+
+	stages{
+		stage('Check upstream builds succeeded'){
+			steps{
+				script{
+					def addLinksDownloadStatusUrl = httpRequest authentication: 'jenkinsKey', url: "${env.JENKINS_JOB_URL}/job/${env.RELEASE_NUMBER}/job/AddLinks-Download/lastBuild/api/json"
+					def addLinksDownloadJson = new JsonSlurper().parseText(addLinksDownloadUrl.getContent())
+					if(addLinksDownloadJson['result'] != "SUCCESS"){
+						error("Most recent AddLinks-Download build status: " + addLinksDownloadJson['result'] + ". Please complete a successful build.")
+					}
+			    	}
+				script{
+					def orthoinferenceDownloadStatusUrl = httpRequest authentication: 'jenkinsKey', url: "${env.JENKINS_JOB_URL}/job/${env.RELEASE_NUMBER}/job/Orthoinference/lastBuild/api/json"
+					def orthoinferenceJson = new JsonSlurper().parseText(orthoinferenceUrl.getContent())
+					if(orthoinferenceJson['result'] != "SUCCESS"){
+						error("Most recent Orthoinference build status: " + addLinksDownloadJson['result'] + ". Please complete a successful build.")
+					}
+			    	}
+		    	}
+	    	}
+		stage('Setup: Backup DB'){
+			steps{
+				script{
+					dir('add-links'){
+						withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
+							sh "mysqldump -u$user -p$pass ${env.RELEASE_CURRENT} > ${env.RELEASE_CURRENT}_${env.RELEASE_NUMBER}_before_addlinks.dump"
+							sh "gzip -f ${env.RELEASE_CURRENT}_${env.RELEASE_NUMBER}_before_addlinks.dump"
+						}
+					}
+				}
+			}
+		}
+		stage('Setup: Build jar file'){
+            		steps{
+		        	script{
+                    			dir('add-links'){
+						sh "mvn clean package -DskipTests"
+                    			}
+          	    		}
+            		}
+        	}
+		stage('Main: AddLinks Insertions'){
+			steps{
+				script{
+					dir('add-links'){
+						withCredentials([file(credentialsId: 'Config', variable: 'FILE')]){
```

Comment:
Similar to above - a change to the `FILE` variable name to be more specific?

Reply:
Yea, I thing in some files I have the variable name changed to ConfigFile. I'll make that uniform across the Jenkinsfiles